tests/conftest.py

```diff
@@ -248,29 +248,31 @@ def application_docker_image_configs() -> Mapping[str, Dict[str, Any]]:
             "": dict(
                 buildargs={
                     "NODE_RUNTIME_FLAGS": "--perf-prof --interpreted-frames-native-stack",
+                    "NODE_IMAGE": "node",
                     "NODE_IMAGE_TAG": "node@sha256:59531d2835edd5161c8f9512f9e095b1836f7a1fcb0ab73e005ec46047384911",
                 }
             ),
             "without-flags": dict(
                 buildargs={
                     "NODE_RUNTIME_FLAGS": "",
+                    "NODE_IMAGE": "node",
                     "NODE_IMAGE_TAG": "node@sha256:59531d2835edd5161c8f9512f9e095b1836f7a1fcb0ab73e005ec46047384911",
                 }
             ),
-            "10-glibc": dict(buildargs={"NODE_IMAGE_TAG": "10-slim"}),
-            "10-musl": dict(buildargs={"NODE_IMAGE_TAG": "10.24.1-alpine"}),
-            "11-glibc": dict(buildargs={"NODE_IMAGE_TAG": "11-slim"}),
-            "11-musl": dict(buildargs={"NODE_IMAGE_TAG": "11-alpine"}),
-            "12-glibc": dict(buildargs={"NODE_IMAGE_TAG": "12.22.12-slim"}),
-            "12-musl": dict(buildargs={"NODE_IMAGE_TAG": "12.22.12-alpine"}),
-            "13-glibc": dict(buildargs={"NODE_IMAGE_TAG": "13-slim"}),
-            "13-musl": dict(buildargs={"NODE_IMAGE_TAG": "13-alpine"}),
-            "14-glibc": dict(buildargs={"NODE_IMAGE_TAG": "14-slim"}),
-            "14-musl": dict(buildargs={"NODE_IMAGE_TAG": "14-alpine"}),
-            "15-glibc": dict(buildargs={"NODE_IMAGE_TAG": "15-slim"}),
-            "15-musl": dict(buildargs={"NODE_IMAGE_TAG": "15-alpine"}),
-            "16-glibc": dict(buildargs={"NODE_IMAGE_TAG": "16-slim"}),
-            "16-musl": dict(buildargs={"NODE_IMAGE_TAG": "16-alpine"}),
+            "10-glibc": dict(buildargs={"NODE_IMAGE": "node", "NODE_IMAGE_TAG": "10-slim"}),
+            "10-musl": dict(buildargs={"NODE_IMAGE": "node", "NODE_IMAGE_TAG": "10.24.1-alpine"}),
+            "11-glibc": dict(buildargs={"NODE_IMAGE": "node", "NODE_IMAGE_TAG": "11-slim"}),
+            "11-musl": dict(buildargs={"NODE_IMAGE": "node", "NODE_IMAGE_TAG": "11-alpine"}),
+            "12-glibc": dict(buildargs={"NODE_IMAGE": "centos/nodejs-12-centos7", "NODE_IMAGE_TAG": "12"}),
```

Comment:
The reason for this test is - to ensure that the nodejs module is successfully loaded on glibc 2.17. I suggest we also add something in the test that "proves" this assumption - e.g in case the base image is changed in the future, and the new image doesn't have this glibc version. For example, a similar check is done here - https://github.com/Granulate/gprofiler/blob/5b82e9bd58ff412f9ed834cd1a423bb9888b2117/tests/test_java.py#L162 the test wants to run on a musl-based image, and the test actually verifies it in runtime. You can check in the test in the 12-glibc case, that `ldd --version` in the container gives 2.17